src/osg/Image.cpp

```diff
@@ -718,14 +721,14 @@ unsigned int Image::computePixelSizeInBits(GLenum format,GLenum type)
 
     switch(format)
     {
-        case(GL_COMPRESSED_RGB_S3TC_DXT1_EXT): return 4;
-        case(GL_COMPRESSED_RGBA_S3TC_DXT1_EXT): return 4;
-        case(GL_COMPRESSED_RGBA_S3TC_DXT3_EXT): return 8;
-        case(GL_COMPRESSED_RGBA_S3TC_DXT5_EXT): return 8;
-        case(GL_COMPRESSED_SIGNED_RED_RGTC1_EXT): return 4;
-        case(GL_COMPRESSED_RED_RGTC1_EXT):   return 4;
-        case(GL_COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT): return 8;
-        case(GL_COMPRESSED_RED_GREEN_RGTC2_EXT): return 8;
```

Comment:
Where'd these RGTC formats go?

Reply:
That was an unintentional removal when fixing a conflict. I'll fix up the pull request.

Reply:
Fixed